src/mixins/common-mixin.ts

```diff
@@ -5,26 +5,17 @@ import { omit, isEqual } from 'lodash'
  * Mixin that provides some useful object utilities.
  */
 @Component({})
-export default class ObjectMixin extends Vue {
-  /**
-   * Removes the specified property from an object.
-   * @param baseObj the base object
-   * @param prop    the property to be removed
-   */
-  omitProp (baseObj: object, prop: Array<string>): object {
-    return omit(baseObj, prop)
-  }
-
+export default class CommonMixin extends Vue {
   /**
    * Removes the specified properties from nested objects.
    * @param baseObj the base object
    * @param keys    the nested object keys which to omit properties from
    * @param prop    the properties to be removed
    */
   omitProps (baseObj: object, keys: Array<string>, prop: Array<string>): object {
-    let parsedObj = {}
-    Object.keys(baseObj).forEach(keys => {
-      parsedObj[keys] = omit(baseObj[keys], prop)
+    const parsedObj: object = {}
```

Comment:
`parsedObj` is never reassigned so it's a const. (We do add properties below but that's different.)